src/main/java/httpserver/HttpServer.java

```diff
@@ -32,7 +35,8 @@
     private HttpRouter router;
 
     private boolean running = true;
-
+    
+    private Logger logger = Logger.getLogger("java-httpserver");
```

Comment:
This should probably be configurable, so someone could use their own logging prefix.